dockers/base-cuda/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ARG UBUNTU_VERSION=20.04
-ARG CUDA_VERSION=11.3.1
+ARG CUDA_VERSION=11.6.1
```

Comment:
The cuda image build is blocked by the lack of a bagua wheel for cuda 11.6.1

`#13 0.911 ERROR: No matching distribution found for bagua-cuda116==0.9.0`

Maybe bagua should still use the old cuda version? Or is there a plan to release an updated one? cc @wangraying

Reply:
According to their docs, `bagua-cuda115` should work with all `CUDA_VERSION >= 11.5`. I tried it and it seems to build